www/src/components/account/AccountAttributes.tsx

```diff
@@ -81,6 +81,7 @@ function DomainMapping({
               onClick={() => {
                 setConfirm(true)
               }}
+              // @ts-expect-error
```

Comment:
Why is this comment needed? A similar comment recently caused the frontend release build to fail and needed to be removed.

Reply:
I made sure the build was passing. Going with honorable v1 made the typing system stricter.